utils/utils_test.go

```diff
@@ -386,7 +386,7 @@ func TestInitPaths(t *testing.T) {
 }
 
 func TestPartialPath(t *testing.T) {
-	path := CreatePartialPath("https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-latest-x86_64")
+	path, _ := CreatePartialPath("https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-latest-x86_64")
```

Comment:
on L395 below, can you add another line testing the etag path as well as the partial path?